curvefs/src/mds/schedule/coordinator.h

```diff
@@ -0,0 +1,181 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * @Project: curve
+ * @Date: 2021-11-8 11:01:48
+ * @Author: chenwei
+ */
+
+#ifndef CURVEFS_SRC_MDS_SCHEDULE_COORDINATOR_H_
+#define CURVEFS_SRC_MDS_SCHEDULE_COORDINATOR_H_
+
+#include <vector>
+#include <map>
+#include <memory>
+#include <string>
+#include <thread>  //NOLINT
+#include "curvefs/src/mds/schedule/operatorController.h"
+#include "curvefs/src/mds/schedule/scheduler.h"
+#include "curvefs/src/mds/schedule/schedule_define.h"
+#include "curvefs/src/mds/topology/topology.h"
+#include "curvefs/src/mds/topology/topology_item.h"
+#include "curvefs/src/mds/schedule/topoAdapter.h"
+#include "curvefs/src/mds/schedule/scheduleMetrics.h"
+#include "src/common/interruptible_sleeper.h"
+
+using ::curvefs::mds::heartbeat::ConfigChangeType;
+using ::curve::common::InterruptibleSleeper;
+
+namespace curvefs {
+namespace mds {
+namespace schedule {
+
+class Coordinator {
```

Comment:
Coordinator can be inherited from ::curve::mds::schedule::Coordinator.

Reply:
That need modify curvebs code. Add todo here, do it later after fs branch merge to master branch.
Add todo here.